### PR TITLE
👔 Set default payout interval for new connected accounts

### PR DESCRIPTION
**Pull request by williamchong**

### CODE REVIEW

File: src/routes/likernft/book/user.ts [hunk 1]
```diff
@@ -147,6 +147,15 @@ router.post(
           metadata: {
             wallet,
           },
+          settings: {
+            payouts: {
+              schedule: {
+                interval: 'monthly',
+                monthly_anchor: 8,
+                delay_days: 14,
```
Comment (nwingt) [inline src/routes/likernft/book/user.ts]: Should be 7?

Comment (williamchong) [reply src/routes/likernft/book/user.ts]: It depends if we expect any refund and charge back

Comment (nwingt) [reply src/routes/likernft/book/user.ts]: then change monthly_anchor to 15?

Comment (williamchong) [reply src/routes/likernft/book/user.ts]: lets do 7 for now then

**Review by nwingt — APPROVED** [review]

**Pull request merged**